app/src/main/java/zapsolutions/zap/fragments/OpenChannelBSDFragment.java

```diff
@@ -0,0 +1,97 @@
+package zapsolutions.zap.fragments;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.FrameLayout;
+import android.widget.ImageButton;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.coordinatorlayout.widget.CoordinatorLayout;
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import com.google.android.material.bottomsheet.BottomSheetDialog;
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+import zapsolutions.zap.R;
+import zapsolutions.zap.lightning.LightningNodeUri;
+import zapsolutions.zap.util.PrefsUtil;
+
+public class OpenChannelBSDFragment extends BottomSheetDialogFragment {
+
+    public static final String TAG = OpenChannelBSDFragment.class.getName();
+    public static final String ARGS_NODE_URI = "NODE_URI";
+
+    private View mNumpad;
+    private EditText mEtAmount;
+    private TextView mTvNodeAlias;
+    private LightningNodeUri mLightningNodeUri;
+    private Button mOpenChannelButton;
+    private View mLocalAmountsInputView;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.bsd_open_channel, container);
+
+        // Apply FLAG_SECURE to dialog to prevent screen recording
+        if (PrefsUtil.preventScreenRecording()) {
+            getDialog().getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);
+        }
+
+        mNumpad = view.findViewById(R.id.Numpad);
+        mEtAmount = view.findViewById(R.id.localAmount);
+        mTvNodeAlias = view.findViewById(R.id.nodeAliasText);
+        mLocalAmountsInputView = view.findViewById(R.id.localAmountInputsView);
+        mOpenChannelButton = view.findViewById(R.id.openChannelButton);
+
+        // temporary coming soon
+        mOpenChannelButton.setText(R.string.coming_soon);
+        mLocalAmountsInputView.setEnabled(false);
+
+        mNumpad.setVisibility(View.VISIBLE);
+        mOpenChannelButton.setEnabled(false);
+        mOpenChannelButton.setTextColor(getResources().getColor(R.color.gray));
+
+        if (getArguments() != null) {
+            mLightningNodeUri = (LightningNodeUri) getArguments().getSerializable(ARGS_NODE_URI);
+            mTvNodeAlias.setText(mLightningNodeUri.getHost() + " (" + mLightningNodeUri.getPubKey() + ")");
```

Comment:
I think if it is possible, we should try to already show the node Alias here. Maybe we could have a little expand box, that would show Ip Address and Pubkey if necessary. But that is something that a normal user should not need to see. It looks to complicated.